src/migration/1539014853555-CreateDatabase.ts

```diff
@@ -0,0 +1,104 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
```

Comment:
All these migration files seem like a bit much for an initial migration, i.e. creating the db for the first time. Lets discuss if there's a better solution.

Reply:
Removed the unnecessary migrations, only added the initial create schema migration. And made changes to the README so it tells that schema:log can be used.